recipes/jupyros/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyros" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyros-{{ version }}.tar.gz
+  sha256: 7f0652a9c4945b2a3adcb0c5dd0e8f82a4cd6754233b19140dbec0f7d69becb3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - jupyter_packaging
+    - pip
+  run:
+    - python >=3.7
+    - ipywidgets >=7.7.2, <8.0
```

Comment:
I think it would be better if you remove the space after the `,` -- not sure what happens if there is an extra space (probably nothing bad, but who knows) :)